lib/Config.ts

```diff
@@ -45,4 +45,6 @@ export default class Config {
     public static readonly LOGO_BW = process.env.NEXT_PUBLIC_LOGO_BW;
 
     public static readonly LOGO_EMAIL = process.env.NEXT_PUBLIC_LOGO_EMAIL;
+    
+    public static readonly REPLY_TO = process.env.SMTP_REPLY_TO || process.env.SMTP_FROM;
```

Comment:
`Config` ist gedacht für Einstellungen die sowohl im Browser als auch Server verfügbar sind, daher auch der `NEXT_PUBLIC_` Prefix.

Reply:
Wenn kein replyTo gesetzt ist, sollte dieser Header auch nicht in der Mail auftauchen.